.evergreen/build_all.sh

```diff
@@ -14,25 +14,14 @@ _cxxflags=""
 
 : "${CONFIGURE_ONLY:=}"
 
-# Use C driver helper script to find cmake binary, stored in $CMAKE.
 if [ "$OS_NAME" = "windows" ]; then
-    : "${CMAKE:=/cygdrive/c/cmake/bin/cmake}"
     # Enable exception handling for MSVC
     _cxxflags="-EHsc"
-    if [ "${WINDOWS_32BIT-}" != "ON" ]; then
+    if test "${WINDOWS_32BIT-}" != "ON" && ! "${USE_NINJA-false}" ; then
```

Comment:
I recommend changing the way the `USE_NINJA` variable is used.  Having the test evaluate the variable contents as a command (`true` or `false` in this case) is slick, but different from how all the other variables in our helper scripts are treated.  I recommend changing the way `USE_NINJA` is handled so that is consistent with other similar variables.